modules/service/src/main/scala/lucuma/odb/graphql/predicate/GmosImagingFilterPredicates.scala

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2016-2025 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.odb.graphql.predicate
+
+import grackle.Path
+import lucuma.odb.data.ObservingModeRowVersion
+
+class GmosImagingFilterPredicates(path: Path):
+  lazy val version: LeafPredicates[ObservingModeRowVersion] =
```

Comment:
what is version in this context?

Reply:
It's the `initial` value vs. the `current` value.  We keep up with some observing mode parameters that were initially selected.  I think this is to facilitate going back after having picked an observing mode in Explore.  If we don't need it, I could simplify the code quite a bit by not tracking the version.

The way this was done originally was to store the initial filters in a separate table, identical to the current version table.  Since I was making the filter table more complex though, with references back to the ETM table, I wanted to keep all the filters for a mode in a single table and `DROP TABLE t_gmos_north_imaging_initial_filter` altogether.  That meant I needed to distinguish the two (`initial` vs. `current`) somehow.